tests/suites/0_stateless/02_function/02_0055_function_strings_humanize.sql

```diff
@@ -0,0 +1,22 @@
+SELECT HUMANIZE_SIZE(1000);
```

Comment:
I think we should modify `let delimiter = 1000_f64;` into `let delimiter = 1024_f64;`.

So:

1. `HUMANIZE_SIZE(1000)` -> `1000.00 B`.
2. `HUMANIZE_SIZE(1024)` -> `1.00 KB`.
3. `HUMANIZE_SIZE(1229);` -> `1.20 KB`

Reply:
1 kB is 1000 bytes (Decimali.e. 10^3 is kilo)
1 KB/KiB is 1024 bytes (Binary i.e. 2^10)

We use `KB` now.

Reply:
> 1 kB is 1000 bytes (Decimali.e. 10^3 is kilo) 1 KB/KiB is 1024 bytes (Binary i.e. 2^10)
> 
> We use `KB` now.

kB/KB/MB always leads to misunderstandings, maybe we should just use the `KiB`.

https://en.wikipedia.org/wiki/Byte#Multiple-byte_units

Reply:
> maybe we should just use the KiB

Ok !

Reply:
@sundy-li updated. PTAL 🙏